seguridad/views.py

```
@@ -7,25 +7,22 @@
+                data['resp'] = True
+                data['user'] = user.username
+                return JsonResponse(data, status=200)
             else:
-                data['error'] = 'Login Fallido!, credenciales incorrectas.'
+                data['error'] = 'Login Fallido!, usuario no esta habilitado'
+        else:
+            data['error'] = 'Login Fallido!, credenciales incorrectas.'
 
-        except Exception as e:
-            data['error'] = str(e)
+        data['error'] = str(e)
```

Ruff (0.11.9)

Undefined name `e`

(F821)